samples/dfu/nrf9160/boards/itlm5_nrf9160_ns.conf

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Logging
+CONFIG_CONSOLE=y
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+CONFIG_RTT_CONSOLE=y
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_LOG=y
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_DFU_TARGET_LOG_LEVEL_DBG=y
+
+# Download target
+CONFIG_DOWNLOAD_HOST="http://176.181.166.45:8000/"
+CONFIG_DOWNLOAD_FILE="app_update_uart.bin"
+#CONFIG_DOWNLOAD_FILE="app_update_uart_wrong_target.bin"
+#CONFIG_DOWNLOAD_FILE="app_update_uart_wrong_magic.bin"
```

Comment:
We typically don't provide invalid test cases in our samples. If we need to validate that these files fail there should be a twister test in the `tests` directory.